### PR TITLE
Added support for Ubuntu 20.04 and root usage

### PR DESCRIPTION
**Pull request by iamarshsingh**
Hi, Adding support for Ubuntu 20.04 and also allow script to work in root environment as by default bower and npm faces installation issues in sudo.
Tested changes in docker image of Ubuntu 20.04, 18.04 and 16.04.

Fixes issue #1864 

Please let me know if any changes are required.

### CODE REVIEW

File: local/install_deps_linux.bash [hunk 1]
```diff
@@ -183,8 +184,8 @@ fi
 if [ ! $only_reproduce ]; then
   # Install other dependencies (e.g. bower).
   nodeenv -p --prebuilt
-  npm install -g bower polymer-bundler
-  bower install
+  npm install --unsafe-perm -g bower polymer-bundler
+  bower --allow-root install
```
Comment (iamarshsingh) [inline local/install_deps_linux.bash:188]: These two flags allowed to support installation in root mode, they are not generally recommended. Can be removed too if we make it explicit to not run install deps as a root user.

Comment (inferno-chromium) [reply local/install_deps_linux.bash:188]: Add a comment like "# Allow bower and polymer-bundler install for root users as well."

Comment (iamarshsingh) [reply local/install_deps_linux.bash:189]: Sure!

**Comment by inferno-chromium** [comment]
Thanks for the super quick fix!

**Pull request merged**